src/shipit_frontend/src/index.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react';
-import { render } from 'react-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import 'font-awesome/css/font-awesome.min.css';
+
+import React from 'react';
+import raven from 'raven-js';
+import { render } from 'react-dom';
 import App from './App';
+import {
+  BACKEND_URL,
+  RELEASE_CHANNEL,
+  RELEASE_VERSION,
+  SENTRY_DSN,
+} from './config';
 
 const root = document.getElementById('root');
-const load = () => render(<App />, root);
+const load = () => {
+  render(
+    <App
+      backend_url={BACKEND_URL}
+      release_version={RELEASE_VERSION}
+      release_channel={RELEASE_CHANNEL}
+    />,
+    root,
+  );
+};
 
-load();
+// When SENTRY_DSN is configured send logs to Sentry
+if (SENTRY_DSN != null) {
+  raven
+    .config(
+      SENTRY_DSN,
```

Comment:
do we need to add anything to the CSP headers to avoid browsers blocking sentry reports?

Reply:
I don't think so, but maybe we should check once we have this in staging. that sentry messages are coming in